vignettes/intro.Rmd

```diff
@@ -27,6 +28,238 @@ uirender(uiinput(icon("dog"),
 ), width = "200px", height = "100px")
 ```
 
+## Create a Shiny App
+Let's create a Shiny app with `semanticPage()` and just a `h1` title.
 
-To define a new page use:
+```{r echo=TRUE, include=TRUE}
```

Comment:
It applied everywhere, unfortunatelly in the current setting we can't run this markdown as shiny app as we parse vignettes to html for documentation, thus `eval = FALSE` is necessary.

Reply:
done and added screen shots